### ISSUE TITLE
"Layer" on OHM website

### ISSUE THREAD

**Issue by c933103** [OpenHistoricalMap/ohm-website#33]
As far as I understand, OHM allow inclusion of features from different times. However on OHM website in the "layer" button there's only a single "Historical" layer button. Shouldn't there be a way for users to choose the years and time to display?

**Comment by bertdeb** [comment]
Yes, there should be a way to filter dates. Some work is being done in this regard:
https://openhistoricalmap.github.io/openhistoricaltiles/frontpage/

**Comment by c933103** [comment]
It would probably be better for the default date selection to be a single date instead of date range as each objects already have its own date range and in the long term when OHM is more completed, using a date range as default would unavoidably causing overlaps in area that are rebuilt within whatever time period that user will select. Using a single date should suffice in most cases, and for visitors who want to see a elapsed change in a single map I guess that can be put in some option or menu next to it.
